src/views/Correction.vue

```diff
@@ -95,9 +95,9 @@ export default class Correction extends Mixins(CommonMixin, DateMixin, FilingTem
     return +this.$route.query['correction-id'] || 0
   }
 
-  /** The filing date, in local timezone. */
-  private get filingDateLocal (): string {
-    return this.convertUtcTimeToLocalTime(this.getFilingDate)?.slice(0, 10)
+  /** The original filing datetime, in Pacific time. */
+  private get originalFilingDate (): string {
+    return this.apiToDateAndTimeString(this.getOriginalFilingDateTime)?.slice(0, 10)
```

Comment:
Bugfix! I noticed that original filing date was today's date (because we didn't set filing date that was used in old code). Fixed now.